sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/ApplicationInsightsRestClient.cs

```diff
@@ -3,46 +3,35 @@
 
 using System;
 using System.Collections.Generic;
-using System.Text.Json;
 using System.Threading;
 using System.Threading.Tasks;
-using Azure;
 using Azure.Core;
 using OpenTelemetry.Exporter.AzureMonitor.Models;
 
 namespace OpenTelemetry.Exporter.AzureMonitor
 {
     internal partial class ApplicationInsightsRestClient
     {
-        internal async Task<Response<TrackResponse>> InternalTrackAsync(IEnumerable<TelemetryItem> body, CancellationToken cancellationToken = default)
+        internal async Task<int> InternalTrackAsync(IEnumerable<TelemetryItem> body, CancellationToken cancellationToken = default)
         {
             if (body == null)
             {
                 throw new ArgumentNullException(nameof(body));
             }
 
             using var message = CreateTrackRequest(body);
+            message.SetProperty("TelemetryItems", body);
             await _pipeline.SendAsync(message, cancellationToken).ConfigureAwait(false);
-            switch (message.Response.Status)
-            {
-                case 200:
-                case 206:
-                case 408:
-                case 429:
-                case 439:
-                case 500:
-                case 502:
-                case 503:
-                case 504:
-                    {
-                        TrackResponse value = default;
-                        using var document = await JsonDocument.ParseAsync(message.Response.ContentStream, default, cancellationToken).ConfigureAwait(false);
-                        value = TrackResponse.DeserializeTrackResponse(document.RootElement);
-                        return Response.FromValue(value, message.Response);
-                    }
-                default:
-                    throw await _clientDiagnostics.CreateRequestFailedExceptionAsync(message.Response).ConfigureAwait(false);
-            }
+
+            return message.TryGetProperty("ItemsAccepted", out var objItemsAccepted) && objItemsAccepted is int itemsAccepted ? itemsAccepted : 0;
+        }
+
+        internal async Task<int> InternalTrackAsync(ReadOnlyMemory<byte> body, CancellationToken cancellationToken = default)
```

Comment:
I see two `InternalTrackAsync` methods.
When would the `IEnumerable` version be used vs the `ReadOnlyMemory`?
As far as I can tell, the `AzureMonitorTransmitter` will only ever call the `IEnumerable` method.

Consider adding a `/// <summary` to both of these methods to help explain.

Reply:
Very good call out, I will get the summary added for these method.

Reply:
`InternalTrackAsync` with `ReadOnlyMemory<byte>` is used to send data from storage directly. We will avoid serialization overhead with this overload. Added summary explaining it.